catalog/src/main/kotlin/com/adevinta/spark/catalog/util/Url.kt

```diff
@@ -40,7 +40,7 @@ public const val DocsUrl: String = "https://adevinta.github.io/spark-android"
 public const val SourceUrl: String = "https://github.com/adevinta/spark-android"
 public const val SparkSourceUrl: String = "https://github.com/adevinta/spark-android/tree/main/spark/src/main"
 
-// Use the real sample url from spark once we have our first ones
+public const val SparkSampleSourceUrl: String = "https://github.com/adevinta/spark-android/blob/main/catalog/src/main/kotlin/com/adevinta/spark/catalog/examples/samples"
```

Comment:
This sample url is not correct, the samples are the one that we want to use in the documentation with the `@sample` identifier

Reply:
What's the correct link?
as I tested it and it's working fine
`https://github.com/adevinta/spark-android/blob/bottom_sheet_component/catalog/src/main/kotlin/com/adevinta/spark/catalog/examples/samples/bottomsheet/BottomSheetExamples.kt`